### ISSUE TITLE
Payload transferred in server side rendering

### ISSUE THREAD

**Issue by ankitshah2787** [chenglou/RCSS#66]
Correct me if I am wrong. If my app is doing server side rendering, I am going to transfer the style string that I get from getStylesString and also I am going to transfer browserify bundle of my components which would also have all the styles that I declared in JS. Am I not transferring same data twice now?

**Comment by jaredly** [comment]
yes. you're also sending the page data twice (once rendered, once with the javascript)
